Add Chinese localized translation

VenenoSix24 opened the pull request:
Hello author, I am a Chinese user. 
I have added **Chinese translation** for this project.
Thank you,good day!

VenenoSix24 committed:
Add Chinese localized translation

VenenoSix24 committed:
更新 master.yml

dante4rt commented:
> Hello author, I am a Chinese user. I have added **Chinese translation** for this project. Thank you,good day!

Thanks for the PR!

However, I noticed that you changed the default data inside `.github/workflows/master.yml`. Could you please revert it to the default? Thanks!

VenenoSix24 committed:
revert master.yml

VenenoSix24 committed:
更新 README.md

VenenoSix24 committed:
Rename README-zh-CN.md to README-CN.md

VenenoSix24 commented:
> > Hello author, I am a Chinese user. I have added **Chinese translation** for this project. Thank you,good day!
> 
> Thanks for the PR!
> 
> However, I noticed that you changed the default data inside `.github/workflows/master.yml`. Could you please revert it to the default? Thanks!

Sry,I misread these two repositories with the same name, I've reverted `master.yml`. 
Thanks for the reminder！

dante4rt commented:
> > > Hello author, I am a Chinese user. I have added **Chinese translation** for this project. Thank you,good day!
> > 
> > 
> > Thanks for the PR!
> > However, I noticed that you changed the default data inside `.github/workflows/master.yml`. Could you please revert it to the default? Thanks!
> 
> Sry,I misread these two repositories with the same name, I've reverted `master.yml`. Thanks for the reminder！

LGTM!
